lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -180,11 +180,7 @@ instance Wellformed (PathParam (ApiT Address, Proxy ('Testnet 0))) where
         "FHnt4NL7yPY7JbfJYSadQVSGJG7EKkN4kpVJMhJ8CN3uDNymGnJuuwcHmyP4ouZ"
 
 instance Malformed (PathParam (ApiT Address, Proxy ('Testnet 0))) where
-    malformed = first PathParam <$>
-        [ ( "NOT-AN-ADDRESS"
-          , "Unable to decode Address: encoding is neither Bech32 nor Base58."
-          )
-        ]
```

Comment:
The generic malformed tests are no longer able to test errors related to the address encoding since it now depends on the underlying backend. So, negative tests are done in backend-specific tests suites on a per-case base .